src/prelude.rs

```diff
@@ -57,3 +57,5 @@ cfg_if! {
         };
     }
 }
+
+pub use scale::Compact;
```

Comment:
Why reexport `Compact`?

Reply:
The idea was to put `Compact` on the same footing as the other re-exported types in the `prelude`, mostly as a marker that this is a type that `scale-info` expects and respects I guess. What is the point of adding a `use scale::Compact` here?

Reply:
I think his point is about simply not re-exporting it and expecting the user to instead use `Compact` directly from the `parity-scale-codec` crate.

Reply:
> The idea was to put Compact on the same footing as the other re-exported types in the prelude

The purpose of the `prelude` is to provide a common interface for required types between `std` and `no_std` environments. Perhaps `prelude` is not a great name then.

Reply:
Yes the name really is bad (sorry for introducing it :sweat_smile: ) . A better name would be something like `alloc_compat` because that's what it actually is since it is a compatibility layer between `std` and `alloc` Rust crates.

Reply:
> we could even merge both crates in the future. haven't thought this through yet, though.

I've thought about this too. They are quite entangled. But that's a future concern I'd say.

Removed the re-export.